relay/sources/rudder-package/tests/webapp_xml/enable_test1.xml.expected

```diff
@@ -10,5 +12,4 @@
     <Arg>org.eclipse.jetty.servlet.SessionPath</Arg>
     <Arg>/</Arg>
   </Call>
-  <Set name="extraClasspath">extra_jar.jar</Set>
-</Configure>
\ No newline at end of file
+<Set name="extraClasspath">extra_jar.jar</Set></Configure>
```

Comment:
Why no newline before the `</Configure>` here ?

Reply:
Because it was easier to implement, I'll try to improve it tomorrow if I have time.